surplus-deficit/surplusDeficit.js

```diff
@@ -0,0 +1,244 @@
+import * as d3 from 'd3';
+import gChartcolour from 'g-chartcolour';
+
+let rem = 10;
+
+export function draw() {
+    let yScale = d3.scaleLinear();
+    let xScale = d3.scaleTime();
+    let seriesNames = [];
+    let yAxisAlign = 'right';
+    let yAxisHighlight = 0;
+    let frameName;
+    const includeAnnotations = d => (d.annotate !== '' && d.annotate !== undefined); // eslint-disable-line
+    let annotate = false; // eslint-disable-line
+    let colourScale = d3.scaleOrdinal();
+
+    function chart(parent) {
+        const width = window.plotDim.width;
+        const height = window.plotDim.height;
+
+        const area = d3.area()
+            .x(d => xScale(d.data.date))
+            .y0(d => yScale(yAxisHighlight))
+            .y1(d => yScale(d[1]));
+
+        const defs = parent.append('defs');
+
+        defs.append('clipPath')
+            .attr('id', `${frameName}chartMask`)
+            .append('path')
+            .attr('d', area)
+            .style('fill', 'white')
+            .attr('opacity', 0.5);
+
+        parent.append('rect')
+            .attr('x', 0)
+            .attr('y', 0)
+            .attr('width', width)
+            .attr('height', d => yScale(yAxisHighlight))
```

Comment:
If `d` isn't used, don't pass it, use `()` instead.

Reply:
Done